src/zenml/repository.py

```diff
@@ -397,6 +397,8 @@ def _migrate_legacy_repository(
             if a legacy repository configuration was not found at the supplied
             path.
         """
+        from zenml.cli import utils as cli_utils
```

Comment:
As I understood it, we have stuck with the convention to only use the cli_utils within the cli code (src/zenml/cli/...) directly. Anything within the code itself we have stuck with our logger. Might be worth thinking about, if there is a way of either moving these logs into the cli code or to revert to using the zenml logger here

Reply:
Zenml logger looked quite ugly and these warnings can be displayed both from CLI and by running regular user code. I will revert to logging though, I think it makes sense.

Reply:
I used the rich console support directly. Does that work ?

Reply:
Yeah absolutely